manifest.lkml

```diff
@@ -21,18 +21,18 @@ constant: SHEETS_CAMPAIGN_TABLE {
 ## Leave the value empty (value:"") if you're not using the source, otherwise you'll get an error
 
 constant: FADS_DATASET_NAME {
-  value: "PacingDashData"
-  export: override_required
+  value: "FbAds-transfer-dataset"
+  export: override_optional
 }
 
 constant: GADS_DATASET_NAME {
-  value: "GoogleAdsPD"
-  export: override_required
+  value: "GoogleAds-Transfer-Dataset"
+  export: override_optional
 }
 
 constant: GADS_CUSTOMER_ID {
-  value: "4717384083"
-  export: override_required
+  value: "11111111111"
```

Comment:
Is this a default or should it be modified by the user?

Reply:
This should be modified by the user

Reply:
They should replace that value when installing the block from the marketplace. It's a constraint with "override required".